### DISCUSSION TITLE
Create page in contribution guide that lists the files/folders for major parts of the Gatsby code base

### DISCUSSION THREAD

**Discussion by KyleAMathews** [gatsbyjs/gatsby#31216] [Ideas / Feature Requests]
E.g.

- actions
- reducers
- api-runners
- build scripts
- etc.

**Comment by chrismgonzalez** [comment]
Looking into claiming this issue.  Is it still up for grabs?

**Comment by marcysutton** [comment]
Yes it is! Based on the original issue title including "contributing", this seems partially related to #11946. We would definitely welcome a PR, and if you have any questions let us know!

**Comment by haardikdharma10** [comment]
@marcysutton I'd like to take up this issue. Just a few questions. Do I need to edit the gatsby-style-guide.md file? If yes, how should I go about it? Unfortunately, the description of the issue looks a bit incomplete. Would be great if you can help me get started. Thanks!
